apps/poker-roll/src/app/queries/helpers.js

```diff
@@ -85,7 +85,7 @@ export function useUpdateSession() {
       });
     },
     onSuccess: () => {
-      queryClient.invalidateQueries({ queryKey: ['sessions'] });
+      queryClient.invalidateQueries({ queryKey: ['sessions', 'bankroll'] });
```

Comment:
This needs to be two separate calls to `invalidateQueries`, once to invalidate `sessions` and again to invalidate `bankroll`. Right now it'll try to find a query key of `['sessions', 'bankroll']` which doesn't exist.